index.html

```diff
@@ -0,0 +1,329 @@
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section id="contact">
+        <h2>
+          I'm always interested in hearing about new projects, so if you'd like
+          to chat please get in touch.
+        </h2>
+        <form action="https://formspree.io/f/mwkakrqj" method="POST">
+          <input
+            type="text"
+            name="fullname"
+            id="fullname"
+            placeholder="Full name"
+            maxlength="30"
+            required
+          />
+          <input
+            type="email"
+            id="email"
+            name="_replyto"
+            placeholder="Email address"
+            required
+          />
```

Comment:
- [ ] Kindly, stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471).
-------
> |Your desktop design | Your mobile desing|
> |-----------------------|----------------------|
> |![Aziz Mejri - Portfolio (2)](https://user-images.githubusercontent.com/71507900/147424448-500a84ca-61c6-4141-8488-128b1c242495.png)|![rawcdn githack com_Whoistolu_Cr-Portfolio_Project_7bc6f0f6a870bf9fff6846ea0fe7d48b8fefd704_index html(Moto G4)](https://user-images.githubusercontent.com/71507900/147424455-f165ba19-4843-44e6-aceb-62e696dafb29.png)|
-------------
> |Figma desktop design | Figma mobile desing|
> |-----------------------|----------------------|
> |![Microverse-Student-Project-1 – Figma (3)](https://user-images.githubusercontent.com/71507900/147424481-646613d8-a6db-43cd-be7e-bffebb09f655.png)|![Microverse-Student-Project-1 – Figma (2)](https://user-images.githubusercontent.com/71507900/147424496-def2ee9f-74fe-42c9-8ed2-5c985a19fd32.png)|